.ci/pipelines/e2e-tests-kind-k8s-versions.Jenkinsfile

```diff
@@ -109,18 +109,40 @@ pipeline {
                     steps {
                         unstash "source"
                         script {
-                            runTests(lib, failedTests, "kindest/node:v1.18.2", "1.18", "ipv4")
+                            runTests(lib, failedTests, "kindest/node:v1.18.8@sha256:f4bcc97a0ad6e7abaf3f643d890add7efe6ee4ab90baeb374b4f41a4c95567eb", "0.9.0", "ipv4")
                         }
                     }
                 }
-                stage("1.18.2 IPv6") {
+                stage("1.19.1") {
                     agent {
                         label 'eck'
                     }
                     steps {
                         unstash "source"
                         script {
-                            runTests(lib, failedTests, "kindest/node:v1.18.2", "1.18", "ipv6")
+                            runTests(lib, failedTests, "kindest/node:v1.19.1@sha256:98cf5288864662e37115e362b23e4369c8c4a408f99cbc06e58ac30ddc721600", "0.9.0", "ipv4")
+                        }
+                    }
+                }
+                stage("1.20.0 IPv4") {
+                    agent {
+                        label 'eck'
+                    }
+                    steps {
+                        unstash "source"
+                        script {
+                            runTests(lib, failedTests, "kindest/node:v1.20.0@sha256:b40ecf8bcb188f6a0d0f5d406089c48588b75edc112c6f635d26be5de1c89040", "0.9.0", "ipv4")
+                        }
+                    }
+                }
+                stage("1.20.0 IPv6") {
```

Comment:
I am not confident that IPv6 will work, in my tests the kind clusters failed to setup the CNI networking correctly, but I am still looking.

Reply:
Found the issue https://github.com/kubernetes-sigs/kind/commit/60074a9e67ddc8d35d3468ab137358b62a4cf723